TTHAnalysis/python/plotter/susy-interface/cmds/constantin/allFriends.sh

```diff
@@ -0,0 +1,9 @@
+T="/mnt/t3nfs01/data01/shome/cheidegg/o/2016-07-13_ewkskims80X"
```

Comment:
The shell scripts in these directories are commands TO the susy-interface. I'd like to keep some files there (I will remove the unnecessary ones, also I'll rename the directories). Also they are useful as examples of how to run the susy-interface.